modules/storage-persistent-storage-aws-maximum-volumes.adoc

```diff
@@ -7,13 +7,9 @@
 
 By default, {product-title} supports a maximum of 39 EBS volumes attached to one
 node. This limit is consistent with the
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS
-volume limits].
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits]. The volume limit depends on the instance type.
 
-{product-title} can be configured to have a higher limit by setting the
-environment variable `KUBE_MAX_PD_VOLS`. However, AWS requires a particular
-naming scheme
-(link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS
-Device Naming]) for attached devices, which only supports a maximum of 52
-volumes. This limits the number of volumes that can be attached to a node via
-{product-title} to 52.
+[IMPORTANT]
+====
+As a cluster administrator, you must use either in-tree or Container Storage Interface (CSI) volumes and respective storage classes, but never both at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
```

Comment:
Would it be helpful for further clarification to say, "you must use either in-tree or Container Storage Interface (CSI) volumes and _their_ respective storage classes, but never both _volume types_ at the same time." ?

Reply:
Yes, that's better!